### DISCUSSION TITLE
提供更多的公共样式

### DISCUSSION THREAD

**Discussion by timongh** [the1812/Bilibili-Evolved#3369] [功能建议 / Ideas]
可能有用的公共样式：

- 主题色色值
- 背景色色值
- border-radius
- 动画/过度时长
- 控件边框样式

暂时只想到这些。另外，某些样式需要用动态方式提供，即 css 自定义变量、js 变量。这样插件才有修改的余地（典型的如夜间模式）。如果提供了 js 变量，组件还能对该颜色进行一定程度的修改（color 库）。

**Comment by the1812** [comment]
除了控件边框, `_common.scss` 里都有提供
